### PR TITLE
[Reader] #2115 Error state for loading document list

### PR DESCRIPTION
**Pull request by NickHeiner**
image

To test, make the request to `documents` fail, and see the error message. Click on the "refresh the page" link and see that the page is refreshed.

Connect #2115.

### CODE REVIEW

File: client/app/components/StatusMessage.jsx [hunk 1]
```diff
@@ -32,15 +33,18 @@ export default class StatusMessage extends React.Component {
...
-      cf-app-segment--alt">
+    return <div id="certifications-generate" className="cf-app-msg-screen cf-app-segment cf-app-segment--alt">
       <h1 className={getClassNames()}>{title}</h1>
-      {leadMessageList.map((listValue, i) =>
-        <h2 className="cf-msg-screen-deck" key={i}>
-          {listValue}
-        </h2>)
-      }
 
+      { children ?
```
Comment (mmlumba) [inline client/app/components/StatusMessage.jsx]: Nice refactoring!

File: client/app/reader/LoadingScreen.jsx [hunk 2]
```diff
@@ -35,14 +36,21 @@ export class LoadingScreen extends React.Component {
...
   render() {
     if (this.props.documentsLoaded) {
       return this.props.children;
     }
 
+    if (this.props.initialDataLoadingFail) {
+      return <StatusMessage
+        title="Unable to load documents">
+        It looks like Caseflow was unable to load this case. Please <a href="">refresh the page</a> and try again.
+        </StatusMessage>;
```
Comment (mmlumba) [inline client/app/reader/LoadingScreen.jsx]: Why is this link blank?

Comment (NickHeiner) [inline client/app/reader/LoadingScreen.jsx]: What do you mean?

Comment (mmlumba) [inline client/app/reader/LoadingScreen.jsx]: There's nothing in the `href` attribute

Comment (NickHeiner) [inline client/app/reader/LoadingScreen.jsx]: Ah, I understand.

If you have an `href` that doesn't start with a `/`, then it's a relative link. So if you're on `/bar`, and you click on `href="foo"`, you go to `bar/foo`. 

A blank `href` doesn't start with a `/`, so it's a relative link. And it doesn't have any additional content, so it takes you back to the page you're presently on. So if you're on `/bar`, and you click on `href=""`, then it refreshes the current page, which is what we're trying to do.

**Pull request merged**